src/armodel/models/port_prototype.py

```diff
@@ -226,6 +228,16 @@ def _validateRPortComSpec(self, com_spec: RPortComSpec):
             pass
         elif isinstance(com_spec, ModeSwitchReceiverComSpec):
             pass
+        elif isinstance(com_spec, ParameterRequireComSpec):
+            if com_spec.parameter_ref == None:
+                raise ValueError(
+                    "The parameter reference of ParameterRequireComSpec has not been defined.")
+            if com_spec.parameter_ref.dest != "PARAMETER-DATA-PROTOTYPE":
+                raise ValueError(
+                    "Invalid parameter dest of ParameterRequireComSpec.")
+            if com_spec.init_value == None:
```

Comment:
init_value is not required. Maybe we can remove this validation.

Reply:
I just analysis the arxml I have. If init_value is not always necessary for ParameterRequireComSpec in standard template, validation could be removed.